src/plugins/StatusBar/StatusBar.js

```diff
@@ -150,12 +154,22 @@ const ProgressBarComplete = ({ totalProgress }) => {
   `
 }
 
-const ProgressBarError = ({ error }) => {
+const ProgressBarError = (props) => {
+  // const { error } = props
   return html`
     <div class="UppyStatusBar-content">
-      <span>
-        ${error.message}
-      </span>
+      <div title="Error">
+        <button title="Retry upload" aria-label="Retry upload" class="UppyStatusBar-action" type="button" onclick=${props.retryAll}>
+          <svg class="UppyIcon" width="28" height="31" viewBox="0 0 16 19" xmlns="http://www.w3.org/2000/svg">
+            <path d="M16 11a8 8 0 1 1-8-8v2a6 6 0 1 0 6 6h2z"/>
+            <path d="M7.9 3H10v2H7.9z"/><path d="M8.536.5l3.535 3.536-1.414 1.414L7.12 1.914z"/><path d="M10.657 2.621l1.414 1.415L8.536 7.57 7.12 6.157z"/>
+          </svg>
+        </button>
+        Upload failed. 
+        <button class="UppyStatusBar-button" type="button" onclick=${props.retryAll}>
```

Comment:
Can we squeeze the `error.message` in here somewhere stll?

Also `"Upload failed."` and `"Retry?"` should probably be localisable (I guess the StatusBar doesn't localise anything so far)

Reply:
Yes, I tend to forget about localization often, will change that, thx! 

I was thinking of placing the error message in a question mark bubble, since most people shouldn’t be concerned with `Error: Failed because: tus: failed to resume upload, caused by [object XMLHttpRequestProgressEvent], originated from request (response code: 0, response text: )` :-D

Reply:
Gotchaa. that works for me!

Reply:
Done.